recipes/ecsimplesnow/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set version = "1.0" %}
+
+package:
+  name: ecsimplesnow
+  version: {{ version }}
+
+source:
+  url: https://github.com/permamodel/Snow_BMI_Fortran/archive/v{{ version }}.tar.gz
+  sha256: f76ad787e9b57792fd2ffd7fd04bdc116b5cbbe31f8303ab326e98d7ff65afc9
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+    - cmake
+
+test:
```

Rather than relying on `curl` and the master branch I’d suggest using `source_files`:
```suggestion
test:
  source_files:
    - data
```

---

@chrisburr Awesome! I hadn't used `source_files` before. I'll update the recipe with this change.